tools/commands-packages-query.js

```diff
@@ -0,0 +1,1159 @@
+    return 1;
+  }
+
+  // Do not return true on broken packages, unless requested in options.
+  var filterBroken = function (match, isRelease, name) {
+    // If the package does not match, or it is not a package at all or if we
+    // don't want to filter anyway, we do not care.
+    if (!match || isRelease)
+      return match;
+    var vr;
+    if (!options["show-all"]) {
+      // If we can't find a version in the local catalog, we want to get the
+      // latest mainline (ie: non-RC) version from the official catalog.
+      vr = projectContext.localCatalog.getLatestVersion(name) ||
+        catalog.official.getLatestMainlineVersion(name);
+    } else {
+      // We want the latest version of this package, and we don't care if it is
+      // a release candidate.
+      vr = projectContext.layeredCatalog.getLatestVersion(name);
+    }
+    if (!vr) {
+      return false;
+    }
+    // If we did NOT ask for unmigrated packages and this package is unmigrated,
+    // we don't care.
+    if (!options["show-all"] && vr.unmigrated){
+      return false;
+    }
+    // If we asked for debug-only packages and this package is NOT debug only,
+    // we don't care.
+    if (options["debug-only"] && !vr.debugOnly) {
+      return false;
+    }
+    return true;
+  };
+
+  if (options.maintainer) {
+    var username =  options.maintainer;
+    // In the future, we should consider checking this on the server, but I
+    // suspect the main use of this command will be to deal with the automatic
+    // migration and uncommon in everyday use. From that perspective, it makes
+    // little sense to require you to be online to find out what packages you
+    // own; and the consequence of not mentioning your group packages until
+    // you update to a new version of meteor is not that dire.
+    selector = function (name, isRelease) {
+      var record;
+      // XXX make sure search works while offline
+      if (isRelease) {
+        record = catalog.official.getReleaseTrack(name);
+      } else {
+        record = catalog.official.getPackage(name);
+      }
+      return filterBroken((name.match(search) &&
+        !!_.findWhere(record.maintainers, {username: username})),
+        isRelease, name);
+    };
+  } else {
+    selector = function (name, isRelease) {
+      return filterBroken(name.match(search),
+        isRelease, name);
+    };
+  }
+
+  buildmessage.enterJob({ title: 'Searching packages' }, function () {
+    _.each(allPackages, function (pack) {
+      if (selector(pack, false)) {
+        var vr;
+        if (!options['show-all']) {
+          vr =
+            projectContext.localCatalog.getLatestVersion(pack) ||
+            catalog.official.getLatestMainlineVersion(pack);
+        } else {
+          vr = projectContext.layeredCatalog.getLatestVersion(pack);
+        }
+        if (vr) {
+          matchingPackages.push({
+            name: pack,
+            description: vr.description,
+            latestVersion: vr.version,
```

Comment:
Yes -- do you think that they shouldn't be?

(mmm: ~/meteor/meteor: mmm search --ejson iron:router
{                                             
  "packages": [
    {
      "name": "iron:router",
      "description": "Routing specifically designed for Meteor",
      "latestVersion": "1.0.6",
      "lastUpdated": {
        "$date": 1418959417958
      }
    }
  ],
  "releases": []
})